meson.build

```
@@ -36,6 +36,7 @@ prefixdir = get_option('prefix')
 
 conf.set_quoted('CONFIG_H_SYSCONFDIR', join_paths(prefixdir, get_option('sysconfdir')))
 conf.set_quoted('CONFIG_H_DATADIR', join_paths(prefixdir, get_option('datadir')))
+conf.set_quoted('PROJECT_VERSION', '0.5.0')
```

```suggestion
conf.set_quoted('HIRTE_VERSION', '0.5.0')
```
Nit: I'd use the actual project name as prefix

---

I'd align with other names in config.h. And of course the version should be read from `meson.build`.

```
conf.set_quoted('CONFIG_H_PROJECT_VERSION', meson.project_version())
```